modules/common-transformer-stream/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/stream/common/sinks/BadSink.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021-2023 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.rdbloader.transformer.stream.common.sinks
+
+import com.snowplowanalytics.snowplow.rdbloader.common.cloud.Queue
+
+sealed trait BadSink[F[_]]
+
+object BadSink {
+  final case class UseBlobStorage[F[_]]() extends BadSink[F]
+  final case class UseQueue[F[_]](queueProducer: Queue.ChunkProducer[F]) extends BadSink[F]
```

Comment:
```suggestion
  final case class BlobStorage[F[_]]() extends BadSink[F]
  final case class Queue[F[_]](queueProducer: Queue.ChunkProducer[F]) extends BadSink[F]
```
Why adding `Use` ?

Reply:
It's just...it sounds a bit better to me? I don't know. Cause if you have raw `BlobStorage` or `Queue` it sounds like specific resources for storage/queue but these are defined somewhere else and provide specific methods we can use.

Here it's more about defining instructions to USE specific resources. So use provided blob storage or use provided queue producer. But it's really a detail and if it looks weird to you I'm ok with changing it!

Reply:
I get your idea of adding action verbs to "resources" that actually do something.

As I see it we always `Use` what we use, but I'm fine with leaving it like this, indeed that's a detail and that's not very important :smile: